config/confd/conf.d/env.toml

```diff
@@ -50,18 +49,20 @@ keys = [
 	"TOKEN_AUTH_CERT_KID",
 	"TOKEN_AUTH_CERT_PUB",
 	"TOKEN_AUTH_JWT_ALGO",
+	"TRUST_PROXY",
+	"VPN_CONNECT_PROXY_PORT",
+	"VPN_GUEST_API_KEY",
 	"VPN_HOST",
 	"VPN_PORT",
 	"VPN_SERVICE_API_KEY",
-	"VPN_GUEST_API_KEY",
-	"VPN_CONNECT_PROXY_PORT",
+	"VPN_UDP_HOST",
+	"WEBRESOURCES_CLOUDFRONT_HOST",
+	"WEBRESOURCES_CLOUDFRONT_PRIVATEKEY_PATH",
+	"WEBRESOURCES_CLOUDFRONT_PUBLICKEY",
 	"WEBRESOURCES_S3_ACCESS_KEY",
-	"WEBRESOURCES_S3_SECRET_KEY",
-	"WEBRESOURCES_S3_REGION",
-	"WEBRESOURCES_S3_HOST",
 	"WEBRESOURCES_S3_BUCKET",
+	"WEBRESOURCES_S3_HOST",
 	"WEBRESOURCES_S3_MAX_FILESIZE",
-	"WEBRESOURCES_CLOUDFRONT_PUBLICKEY",
-	"WEBRESOURCES_CLOUDFRONT_HOST",
-	"WEBRESOURCES_CLOUDFRONT_PRIVATEKEY_PATH"
+	"WEBRESOURCES_S3_REGION",
+	"WEBRESOURCES_S3_SECRET_KEY",
```

Comment:
Can you do this reordering in a separate commit? It's nice to have it done, but it makes it much harder to review.

Reply:
They should actually just be removed altogether as they have no purpose when using an env var backend

Reply:
Done, deleted.